src/console_op/data.go

```diff
@@ -34,13 +34,34 @@ import (
 	"log"
 	"net/http"
 	"time"
+
+	"github.com/go-chi/chi/v5"
 )
 
 // Variable to hold address of console-data service
 var dataAddrBase string = "http://cray-console-data/v1"
 
+type DataService interface {
```

Comment:
Why have a separate DataService interface?  I don't see anything else that implements it other than DataManager.

Reply:
That is correct, it is to allow loose coupling with the concrete implementations so we can provide a mock struct in testing. Also if the interface changes then the compiler will let us know that a `manager` technically is not of type `service` meaning we will need to implement the new method.